recipes/qpoases/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
As you're patching the version I'd suggest validating it if you can:
```suggestion
# Ensure the version is correct in the sources
grep 'PACKAGE_VERSION "'$PKG_VERSION'"' CMakeLists.txt

```

Reply:
Thanks, this would be useful also to check that in future the version is correctly updated.